src/main/java/project/console/ConsoleUI.java

```diff
@@ -0,0 +1,60 @@
+package project.console;
+
+import project.service.LibraryService;
+
+import java.util.Scanner;
+
+public class ConsoleUI {
+    private LibraryService libraryService = new LibraryService();
+
+    Scanner scanner = new Scanner(System.in);
+
+    public void run() {
+        System.out.println("Welcome to the library!");
+
+        while (true) {
+            getCover();
+        }
+    }
+
+    private void getCover() {
+        getMessage();
+        String choice = getChoice();
```

Comment:
Тут у тебя лишний код, внутри метода `getChoice()`. В целом - это очень правильный подход - выносить куски кода в отдельные методы. Но там внутри ты просто берешь данные из консоли и конвертируешь в строку.
А можно это делать сразу, за одно действие. И тогда это уже не стоит выносить в отдельный метод.
Вот так это может выглядеть:
```suggestion
        var choice = scanner.nextLine();
```

Reply:
Да, так лучше, исправил